packages/atlas/src/views/studio/MyPaymentsView/PaymentsTransactionTab/utils.ts

```diff
@@ -0,0 +1,41 @@
+import BN from 'bn.js'
```

Comment:
I would name this file `PaymentTransactionTab.utils.ts`. I know that we still have some files called just `utils`, but we decided one day, that if something is "component specific" we're naming this `ComponentName.utils`. We already have `Comment.utils` or `NftForm.utils`